providers/common/messaging/src/airflow/providers/common/messaging/providers/__init__.py

```diff
@@ -16,6 +16,7 @@
 # under the License.
 from __future__ import annotations
 
+from airflow.providers.common.messaging.providers.kafka import KafkaMessageQueueProvider
 from airflow.providers.common.messaging.providers.sqs import SqsMessageQueueProvider
 
-MESSAGE_QUEUE_PROVIDERS = [SqsMessageQueueProvider()]
+MESSAGE_QUEUE_PROVIDERS = [SqsMessageQueueProvider(), KafkaMessageQueueProvider()]
```

Comment:
This is what we would have to do to make it happens - we would have to discover the available providers from the packages installed.